app/src/main/AndroidManifest.xml

```diff
@@ -102,7 +102,8 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.settings.profile.ProfileResetPinActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:title="@string/profile_reset_pin_activity_title" />
```

Comment:
Use `android:label="@string/profile_reset_pin_activity_title"`

instead of `android:title`